h5p-bildetema/src/main.tsx

```diff
@@ -38,7 +38,8 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
             <ContentIdContext.Provider value="content-id">
               <App
                 defaultLanguages={["nob"]}
-                backendUrl="https://cdn-prod-bildetema.azureedge.net/data/database.json.tar.gz"
+                backendUrl="https://cdn-dev-bildetema.azureedge.net/data/data.json.tar.gz"
+                // backendUrl="https://cdn-prod-bildetema.azureedge.net/data/database.json.tar.gz"
```

Comment:
Should the old backend url still be here? Comment if so, else we can store it somewhere else